imagepuller/main.go

```diff
@@ -34,17 +38,19 @@ func main() {
 		return
 	}
 
+	flag.Parse()
```

Comment:
please use cobra for command line interface, checkout the other packages for examples.